elements/lisk-transactions/test/fee.spec.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import { getAddressAndPublicKeyFromPassphrase } from '@liskhq/lisk-cryptography';
+import * as Fee from '../src/fee';
+
+const validAssetSchema = {
```

Comment:
all variable should be within `describe`

Reply:
not necessarily? https://jestjs.io/docs/en/api#describename-fn, and we do use mix of both approach in SDK, so we can keep it this way